main/e2e-testing.md

```diff
@@ -143,3 +141,142 @@ Some examples projects utilizing `@agoric/synpress` for e2e tests:
 
 - [PSM dApp E2E Tests](https://github.com/Agoric/dapp-psm/tree/main/tests/e2e)
 - [Wallet dApp E2E Tests](https://github.com/frazarshad/wallet-app/tree/main/test/e2e)
+
+## @agoric/synpress Commands
+
+The `@agoric/synpress` commands are custom Cypress commands designed for use with Keplr. They provide ready-made functions to interact with Keplr-specific elements and workflows.
+
+### `setupWallet`
+
+Initializes a Keplr wallet for testing. You can either create a new one or import an existing one using the provided options.
+
+**Arguments:**
+
+(All arguments are optional)
+
+The command takes a single object as an argument with the following keys:
+
+- `secretWords` _(**string**)_: The secret words for the wallet. (default is a predefined mnemonic phrase that represents a wallet address on the Agoric chain)
+- `privateKey` _(**string**)_: The private key for the wallet.
+- `password` _(**string**)_: The password for the wallet. (default is `Test1234`).
+- `newAccount` _(**boolean**)_: Whether to create a new wallet or not. (default is `false`).
+- `walletName` _(**string**)_: The name of the wallet. (default is `My Wallet`).
+- `selectedChains` _(**array**)_: The chains to select. (default is an empty array, `[]`).
+- `createNewWallet` _(**boolean**)_: Whether to create a new wallet (default is `false`).
+
+**Returns:**
+
+- _(**boolean**)_: `true` if the wallet was set up successfully, `false` otherwise.
+
+### `acceptAccess`
+
+Handles the authorization process for connecting the Keplr wallet to a DApp. DApps provide a `Connect Wallet` option to initiate this process.
+
+**Returns:**
+
+- _(**boolean**)_: `true` if access was accepted successfully, `false` otherwise.
+
+### `confirmTransaction`
+
+Handles signing transactions within a DApp using the Keplr wallet. This occurs when a user initiates a transaction, such as sending tokens or interacting with a smart contract.
+
+**Returns:**
+
+- _(**boolean**)_: `true` if transaction was approved successfully, `false` otherwise.
+
+### Managing Browser Tabs
```

Comment:
For consistency make this ## and its subheading commands ### or in the previous section do the opposite: make the heading ### and commands ####.